packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -362,62 +343,6 @@ export class GCTelemetryTracker {
 			}
 		}
 	}
-
-	/**
-	 * Log events that are pending in pendingEventsQueue. This is called after GC runs in the summarizer client
-	 * so that the state of an unreferenced node is updated.
-	 */
-	public async logPendingEvents(logger: ITelemetryLoggerExt) {
```

Comment:
I didn't compare this against the code above - was there anything else different in here? Or was it just basically duplicated?